kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -495,6 +495,9 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
     mod_compare_elem_cond_utils.def("GetRegisteredName", GetRegisteredNameElement );
     mod_compare_elem_cond_utils.def("GetRegisteredName", GetRegisteredNameCondition );
 
+    // PropertiesUtilities
+    auto mod_prop_utils = m.def_submodule("PropertiesUtilities");
```

Comment:
why a submodule?
normally we use a class for this.

Reply:
We use submodules for utilities when using namespaces @philbucher suggested some months ago

Reply:
Yep, submodule for namespace and py::class for a cpp-class